public-resources/EvaChecks.csv

```diff
@@ -53,7 +53,8 @@ dq,Error,Incorrect Year Entered Service,V1.1 - Veteran's Information: Year Enter
 dq,Error,Incorrect Year Separated,V1.1 - Veteran's Information: Year Separated from Military Service,The year the client separated from military service is in the future. Please verify the dates this client was in military service to correct this error.,,18
 dq,Error,Invalid Move-In Date,3.20 - Housing Move-In Date,This move-in date does not fall between the Entry Date and the Exit Date or this move-in date is after the Export End Date. Please review and correct.,,40
 dq,Error,Missing Approximate Date Homeless,3.917.3 - Prior Living Situation: Date Homelessness Started,This data element is required to be collected at Project Start. Please go to the client's assessment at Project Start to enter this data to HMIS.,,28
-dq,Error,Missing Destination Subsidy Type,"3.12.1 - Destination, 3.12.A - Destination Subsidy Type","If a client exits to Rental by Client, the user must record the Subsidy Type.",,121
+dq,Error,Missing Current Living Situation Subsidy Type,"4.12 - Current Living Situation, 4.12A Current Living Situation Subsidy Type","If the response to Current Living Situation is ""Rental by client, with ongoing housing,"" then the Rental Subsidy Type must be specified.",,129

```

Comment:
Shouldn't this say "If the response to Current Living Situation is 'Rental by client, with ongoing housing **subsidy**"? Same with the others.